packages/client/src/comp/Contact.tsx

```diff
@@ -97,22 +97,28 @@ export const Contact: React.FC = () => {
     e.persist()  // allow async function call
     e.preventDefault()
     setFetchingData(true)
-    try {
-      await client.contactUs(
-        emailRef.current?.value ?? '',
-        nameRef.current?.value ?? '',
-        textRef.current?.value ?? '',
-      )
-
-      toast(
-        'Your email has successfully been sent to our support team. Expect to hear from us soon.',
-        {type: 'success'}
-      )
-    } catch(e) {
-      toast(
-        'Failed to send contact email',
-        {type: 'error'}
-      )
+    if (emailRef.current && nameRef.current && textRef.current) {
+      try {
+        await client.contactUs(
```

Comment:
Check for the return and if there is an error, write 'We were unable to submit this form.  Please submit again.'